modules/serverless-rn-1-31-0.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-31-0_{context}"]
+= Red Hat {ServerlessProductName} 1.31
+
+{ServerlessProductName} 1.31 is now available. New features, updates, and known issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes.
+
+[id="new-features-1-31-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.10.
+* {ServerlessProductName} now uses Knative Eventing 1.10.
+* {ServerlessProductName} now uses Kourier 1.10.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.10.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.10.
+* The `kn func` CLI plug-in now uses `func` FIXME.
+
+* When specifying Kourier for Ingress and using `DomainMapping`, the TLS for OpenShift Route is set to passthrough, and TLS is handled by the Kourier Gateway. Beginning with {ServerlessProductShortName} 1.31, it is possible to specify the enabled cipher suites on the side of the Kourier Gateway.
+
+* Now you can change the pattern for Apache Kafka topic names used by Knative brokers and channels for Apache Kafka.
+
+* The `domain-mapping` deployment in `knative-serving` has been integrated into the `controller` deployment.
+* The `domain-mapping-webhook` deployment in `knative-serving` has been integrated into the `webhook` deployment.
```

Comment:
Where does this change come from?
I'm not aware of this and looking at deployments from 1.31, I see this:
```
oc get pods -n knative-serving | grep -e domain -e controller
controller-66658b8885-ss5jq                                   2/2     Running     0          4m2s
controller-66658b8885-vxtgt                                   2/2     Running     0          4m19s
domain-mapping-56c6d67d66-82ljb                               2/2     Running     0          4m25s
domain-mapping-56c6d67d66-xjj26                               2/2     Running     0          4m25s
domainmapping-webhook-86c5f8d4df-b4s89                        2/2     Running     0          4m25s
domainmapping-webhook-86c5f8d4df-mk5tj                        2/2     Running     0          4m25s

```

Reply:
@mgencur https://issues.redhat.com/browse/SRVKS-1155

Reply:
I see this https://github.com/knative/serving/pull/14024 and it was rejected, then I see this commit merged only in release-v1.11 branch which is Serverless 1.32: https://github.com/knative/serving/commit/fc166ac8b459a73116ca5535e168aba2d97a2626 ; so only the domainmapping-webhook is merged with the other webhook and that will be available in 1.32.
Let's remove the note from the release notes for now.